src/qt/pivx/forms/addnewaddressdialog.ui

```diff
@@ -113,7 +113,7 @@
         <item>
          <widget class="QLabel" name="labelAddress">
           <property name="text">
-           <string>D7VFR83SQbiezrW72hjcWJtcfip5krte2Z </string>
+           <string>Address</string>
```

Comment:
Might as well change this to `<string notr="true">Address</string>` so it can be ignored by Transifex.

Reply:
yeah ok, will do it.